src/core/execute.c

```diff
@@ -1134,7 +1135,8 @@ void exec_context_dump(const ExecContext *c, FILE* f, const char *prefix) {
                 prefix, protect_hostname_to_string(c->protect_hostname), c->private_hostname ? ":" : "", strempty(c->private_hostname),
                 prefix, protect_proc_to_string(c->protect_proc),
                 prefix, proc_subset_to_string(c->proc_subset),
-                prefix, private_bpf_to_string(c->private_bpf));
+                prefix, private_bpf_to_string(c->private_bpf),
+                prefix, c->bpf_delegate_commands);
```

Comment:
`%s` with `BPF_DELEGATE_TO_STRING(c->bpf_delegate_commands)` should be better?

Reply:
but bpf_delegate_to_string() wants a preallocated buffer, it's not worth it

Reply:
ah I just saw your proposal below